docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -0,0 +1,59 @@
+---
+id: app-crash-logs
+title: App Crash Logs
+sidebar_label: App Crash Logs
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
```

Comment:
Can you add the Beta badge or update the sidebar label with (beta)?

Reply:
Great point